google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineAssetProvider.java

```diff
@@ -65,21 +63,35 @@ public XmlFile loadAppEngineStandardWebXml(@NotNull Project project,
           project, "appengine-web.xml", module.getModuleContentScope()));
     }
 
-    Iterables.filter(appEngineWebXmls, Predicates.notNull());
```

Comment:
We could, though this returns an `Iterator`, and we would still have to convert it into a list immediately afterward in order to sort according to the `AppEngineWebXmlOrdering`. This seemed a bit cleaner to me, though I don't feel that strongly about it.